functional-test/src/test/groovy/org/rundeck/util/common/jobs/JobUtils.groovy

```diff
@@ -112,9 +113,9 @@ class JobUtils {
                 "</joblist>"
     }
 
-    static Job getJobDetailsById(final String jobId, final ObjectMapper mapper, RdClient client){
+    static JobDetails getJobDetailsById(final String jobId, final ObjectMapper mapper, RdClient client){
```

Comment:
Deserialization was failing when the job had the "options" property. 
`Job` expected the `Map`, but a list of object was returned from the API. Had to introduce `JobDetails`